### PR TITLE
#288 Add a concept API subpage for the property-view.

### PR DESCRIPTION
**Pull request by ndoschek**
It gives an overview of the concept as well as an example for a custom property-view contribution.

Resolves https://github.com/eclipse-theia/theia-website/issues/288

Contributed on behalf of STMicroelectronics.

Signed-off-by: Nina Doschek <ndoschek@eclipsesource.com>

### CODE REVIEW

File: src/docs/property_view.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+which shows additional information about the current selection within the IDE.
+Property views are heavily used in those IDEs for showing details of elements in e.g. diagram editors,
+complex tree editors, or the file explorer selection.
+Therefore, the main idea is to have a global, generic property view in the IDE,
+but allow specific implementations to extend the contents of the global property view
+with specific additional information for some type of selection.
+
+The `@theia/property-view extension` contributes a generic, global property view based on Theia's global selection.
+The property view widget can be opened/toggled either via menu `View->Properties` or via shortcut <kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>P</kbd>.
+It is located in the bottom dock area per default.
```
Comment (vince-fugnitto) [inline src/docs/property_view.md:17]: minor - should it be `per` or `by`:

```suggestion
It is located in the bottom dock area by default.
```

File: src/docs/property_view.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+The following two default content widgets are implemented in this extension:
+
+`EmptyPropertyViewWidget`: If no other widget can be provided, a simple message (`No properties available`) is shown.
+`ResourcePropertyViewWidget`: Displays additional information about a file (e.g. location, name, last modified) that is selected in the file explorer or in the active monaco editor.
```
Comment (vince-fugnitto) [inline src/docs/property_view.md:21]: Just to illustrate better that these are the two widgets, we can have them in lists:

```suggestion
- `EmptyPropertyViewWidget`: If no other widget can be provided, a simple message (`No properties available`) is shown.
- `ResourcePropertyViewWidget`: Displays additional information about a file (e.g. location, name, last modified) that is selected in the file explorer or in the active monaco editor.
```

File: src/docs/property_view.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+## Contribute a custom property view
```
Comment (vince-fugnitto) [inline src/docs/property_view.md:23]: In general we try to capitalize in a title if possible:

```suggestion
## Contribute a Custom Property View
```

**Review by vince-fugnitto — COMMENTED** [review]
Overall the docs look good, I just have minor comments :+1:

**Comment by ndoschek** [comment]
Thank you for the review @vince-fugnitto, I addressed your comments and force pushed my commit.

**Review by vince-fugnitto — APPROVED** [review]
LGTM! I don't have any further comments.

**Pull request merged**